pkg/plugin/schema/getter.go

```diff
@@ -0,0 +1,54 @@
+/*
+ Copyright The Helm Authors.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package schema
+
+import (
+	"bytes"
+	"time"
+)
+
+// TODO: can we generate these plugin input/outputs?
+
+type GetterOptionsV1 struct {
+	URL                   string
+	CertFile              string
+	KeyFile               string
+	CAFile                string
+	UNTar                 bool
+	InsecureSkipVerifyTLS bool
+	PlainHTTP             bool
+	AcceptHeader          string
+	Username              string
+	Password              string
+	PassCredentialsAll    bool
+	UserAgent             string
+	Version               string
+	Timeout               time.Duration
+}
+
+type GetterInputV1 struct {
```

Comment:
how about this?

```suggestion
type InputMessageGetterV1 struct {
```

And then also version `type ConfigGetter struct` to `type ConfigGetterV1` struct` (to match plugin type string `getter/v1`. and likewise version all the other ConfigTYPE implementations)?

Reply:
(updated name)